package.json

```diff
@@ -24,7 +24,7 @@
 	},
 	"scripts": {
 		"docs": "npm run docs-theme && npm run docs-browser && npm run docs-server && npm run docs-react-native",
-    "docs-theme": "pushd typedoc-theme && npm run build",
+		"docs-theme": "pushd typedoc-theme && npm run build",
```

Comment:
[nit] ditto

Reply:
You're right. Good catch! But I think this change actually makes it consistent now.